DVR/Cassette.swift

```diff
@@ -13,8 +13,20 @@ struct Cassette {
         for interaction in interactions {
             let r = interaction.request
 
+            let equivalentBody: Bool
+
+            if let rBody = r.HTTPBody,
```

Comment:
Oh, derp. Just noticed we already have `requestBody`... I'm just confused by the naming. What is `r` vs. what is `request`.